hudi-common/src/main/java/org/apache/hudi/common/fs/inline/InLineFSUtils.java

```diff
@@ -29,20 +29,23 @@
 public class InLineFSUtils {
   private static final String START_OFFSET_STR = "start_offset";
   private static final String LENGTH_STR = "length";
+  private static final String PATH_SEPERATOR = "/";
+  private static final String SCHEMA_SEPERATOR = ":///";
```

Comment:
I guess this is called SCHEM**E** and not SCHEM**A**

Reply:
Right, fixed it.